plugin/trino-hive/src/main/java/io/trino/plugin/hive/Schema.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive;
+
+import io.airlift.slice.SizeOf;
+
+import java.util.Map;
+
+import static io.airlift.slice.SizeOf.estimatedSizeOf;
+import static io.airlift.slice.SizeOf.instanceSize;
+import static java.util.Objects.requireNonNull;
+
+public record Schema(
+        String deserializerClassName,
```

Comment:
Actually it is not just `deserializedClassName` right ? It maintains HiveSerDe - which is used for both serialization and deserialization - Can we rename them as a `serializationLibrayName` or `hiveSerde` - to be in sync with Hive's model

Reply:
I was trying to be consistent with the naming of the method `io.trino.plugin.hive.util.HiveUtil#getDeserializerClassName` from where we extract this string.
But yeah `serializationLibrayName` is also fine as that matches `io.trino.plugin.hive.util.SerdeConstants#SERIALIZATION_LIB`